src/components/ProductListItem/ProductListItem.js

```diff
@@ -1,27 +1,38 @@
 import React from 'react'
+import { useHistory } from 'react-router-dom'
 import styled from 'styled-components'
+import Button from '../Button/Button'
+import DividerLine from '../DividerLine/DividerLine'
 
 export default function ProductListItem({ product }) {
```

Comment:
Du könntest dir unten ein wenig Text sparen, indem du das Produkt nochmal destructured reingibst:

`export default function ProductListItem({product: {name, ownerFirstname, location})`
Dann kannst du unten wenn du die Props einfügst, das product. sparen, also bspw.:

```
 return (
<DividerLine />
<h4>{name}</h4> 
<StyledProductInfo>
 <div>von {ownerFirstname}</div>
 ...)
```

In deiner ProductListPage, wo du das ProductListItem aufrufst, kannst du weiterhin einfach nur das Product reingeben

Reply:
Sehr gute Idee :) So mache ich das!